### PR TITLE
Update docs/muted heading page content

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- Updated copy ✏️ 
- Updated demo example 💯 
- Check content matches copy doc [here](https://docs.google.com/document/d/1ZFRd4z7TsU4oifxgky77337v_1fkE2ak12tGPNivD70/edit#heading=h.r9fmkym6ojv6)

## QA
- Pull code
- Run /docs
  - Check http://0.0.0.0:8104/en/patterns/muted-heading
- Pull code
- Run ./run
- Review example 😉 
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/muted-heading/

## Fixes
- Fixes https://github.com/ubuntudesign/design-vanilla-framework/issues/388

### CODE REVIEW

File: docs/en/patterns/muted-heading.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ title: Muted heading
 
 ## Muted heading
 
-A Muted heading can be used to introduce a collection of icons or images.
+<hr>
+
+A muted heading can be used to introduce a collection of icons or images.
```
Comment (deadlight) [inline docs/en/patterns/muted-heading.md]: We use it elsewhere. Perhaps we should rephrase this as something like:

"A muted heading can be used to subtly provide context to content without the heading itself being too prominent"

I'm not sure that that's the best wording.

Comment (kwm14) [reply docs/en/patterns/muted-heading.md]: I'm ok with that, reads better than what I put together 😆 will update.
